server/src/main/proto/spine/server/dispatch/dispatching.proto

```diff
@@ -133,6 +139,7 @@ message Interruption {
 
 // Ignored signal handling.
 message Ignore {
+    option (is).java_type = "IgnoreMixin";
```

Comment:
Let's consider making this a convention and removing the need to add the option if the message type implements an interface named in the same way as the type itself but with the `Mixin` suffix.

If I remember correctly, we already pass the whole classpath to our `protoc` plugin. We might as well check if there is such an interface.

Although, this is, most likely, not a v1.0 feature.

Reply:
Also, generating messages to implement `FieldAwareMessage` is fairly easy at this point.

Reply:
As discussed in person, we would rather invest in a generation of a fully-blown in-place validation.